### PR TITLE
Autocomplete test functional init

### PR DESCRIPTION
**Pull request by kadhikari**
Test functional for places initialized

### CODE REVIEW

File: source/autocomplete/autocomplete_api.cpp [hunk 1]
```diff
@@ -32,6 +32,7 @@ www.navitia.io
 #include "type/pb_converter.h"
 #include "autocomplete/autocomplete.h"
 #include "type/pt_data.h"
+#include "string.h"
```
Comment (antoine-de) [inline source/autocomplete/autocomplete_api.cpp]: the convention is to include outside stuff with '<>'

you are sure you need this include ?

File: source/autocomplete/autocomplete_api.cpp [hunk 2]
```diff
@@ -351,7 +352,7 @@ pbnavitia::Response autocomplete(const std::string &q,
             const auto b_order = get_embedded_type_order(b.embedded_type());
             return  a_order< b_order;
         } else if(a.quality() == b.quality()) {
-            return a.name() < b.name();
+                return boost::algorithm::lexicographical_compare(a.name(), b.name(), boost::is_iless());
```
Comment (antoine-de) [inline source/autocomplete/autocomplete_api.cpp]: indentation problem

File: source/ed/build_helper.cpp [hunk 3]
```diff
@@ -387,4 +387,24 @@ void builder::connection(const std::string & name1, const std::string & name2, f
          }
      }
  }
+
+ void builder::manage_admin() {
```
Comment (antoine-de) [inline source/ed/build_helper.cpp:395]: put a comment here to explain what you are doing.
(eg that only the first admin is used)

File: source/georef/georef.cpp [hunk 4]
```diff
@@ -241,6 +241,16 @@ void GeoRef::add_way(const Way& w){
     ways.push_back(to_add);
 }
 
+void GeoRef::add_admin(const Admin& a){
```
Comment (antoine-de) [inline source/georef/georef.cpp]: If it is only used for test purposes, it's better to move this function in the tests (a free function for example).

That being said, I'm not sure this function is really useful. If you want it to be syntaxic sugar, I think I would be more convenient to put the parameter.

ex:
in the test:

void add_admin(const std::string& name, ...level, .. post_code) {
Admin\* to_add = new Admin;
 to_add->name = name;
 to_add->uri = a.name;
 to_add->post_code = post_code;
 to_add->level = level;
to_add->idx = admins.size();
 admins.push_back(to_add);
}

**Comment by antoine-de** [comment]
Thanks for adding lots of tests!

File: source/autocomplete/autocomplete_api.cpp (hunk 1 above)
Comment (kadhikari) [inline source/autocomplete/autocomplete_api.cpp]: deleted

File: source/autocomplete/autocomplete_api.cpp (hunk 2 above)
Comment (kadhikari) [inline source/autocomplete/autocomplete_api.cpp]: ok

File: source/ed/build_helper.cpp (hunk 3 above)
Comment (kadhikari) [inline source/ed/build_helper.cpp:395]: comments added

File: source/georef/georef.cpp (hunk 4 above)
Comment (kadhikari) [inline source/georef/georef.cpp]: Like add_way, add_housenumber we can certainly use add_admin in other functions.

**Comment by kadhikari** [comment]
Antoine, i posted some corrections

**Comment by kadhikari** [comment]
I deleted add_admin in georef

**Pull request merged**